examples/lstm_seq2seq.py

```diff
@@ -113,7 +113,7 @@
     for t, char in enumerate(input_text):
         encoder_input_data[i, t, input_token_index[char]] = 1.
     for t, char in enumerate(target_text):
-        # decoder_target_data is ahead of decoder_target_data by one timestep
+        # decoder_input_data is ahead of decoder_target_data by one timestep
```

Comment:
Comment is incorrect; it is the target data that is ahead and the input data that is behind. The target is the future. Please fix : )

Reply:
Got it, now fixed with a follow-up commit.